### ISSUE TITLE
Spurious Wavebox notifications with no text 

### ISSUE THREAD

**Issue by CheechGe** [wavebox/waveboxapp#758]
* Wavebox Version: [e.g. 3.0.0]
3.14.10Beta

* Operating System & Version: [e.g. Windows 10, Mac OSX 10.3.4, Ubuntu 17.10]
Windows 10

* (Linux only) Install method: [e.g. Debian installer, Snap Package, Tar, Apt-Repo, etc...]
* Account type (if applicable): [e.g. Gmail, Office 365, Trello, etc...]
N/A

**Expected & actual behavior**
Some information about what you expect to happen and what actually happens.
Occasionally a Wavebox notification message arrives but there is nothing in the text that indicates what the notification was for and the notification message randomly is displayed or not in the notifications panel. 

This happens about 5-6 times a day and while not a major problem it can make one wonder if something was missed.

**Steps to reproduce**
Are there any steps that we can take to reproduce the bug?
None known
**Is the bug persistent or intermittent?**
Do you only see it sometimes or does it always happen?

**Screenshots**
Screenshots often help us narrow down a bug more easily.
Not at this time but will try to capture it if it hangs around.

**Additional information**
No mail message had arrived at the time the notification was displayed, however it is possible that a mail did arrive but was filtered out.

**Comment by nmat** [comment]
hmmm I am guessing that you have a site added that does not allow for notifications to be sent through maybe?

**Comment by CheechGe** [comment]
I guess that is possible.  I haven't added any new sites in a couple of weeks and all of a sudden (at least as I observed) these notifications started appearing.  As of yesterday afternoon I removed the add-on that may have been causing the duplicate email "read" notifications and both have stopped, at least at this time.  I'll watch it over the next few days and see what happens.

**Comment by Thomas101** [comment]
If you click on the Wavebox tray icon in the bottom left of your screen a popup will appear. The `Notifications` tab has a list of the last few notifications that came in. There might be more info here about which tab sent them...

![screen_shot_2018-08-15_at_08_57_03](https://user-images.githubusercontent.com/103586/44137752-7474005a-a069-11e8-9266-723a100b2aed.png)

**Comment by CheechGe** [comment]
I did that.  The first time that occurred  the only notification in the Notifications panel was the exact same Notification message that appeared at the bottom, no indication other than it was a Wavebox Notification as indicated in the issue.

I had closed the Notification window and when I opened it again to do a screen capture it was gone.  The next time one popped up I immediately opened the Notification panel with the intent to do a screen shot but the notification message was not there.

These have't occurred any more since I removed the Gmelius extension along with the multi email Read notifications so I suspect they are related.  I will reinstall the Gmelius extention and see if things start happening again.
